$OpenBSD: patch-src_core_CMakeLists_txt,v 1.9 2015/07/21 10:06:14 landry Exp $
chunk 1/2: dont call bison to generate the parser, and build the pregenerated file instead
--- src/core/CMakeLists.txt.orig	Fri Jun 26 14:38:57 2015
+++ src/core/CMakeLists.txt	Mon Jun 29 22:08:17 2015
@@ -93,6 +93,7 @@ SET(QGIS_CORE_SRCS
   qgsdistancearea.cpp
   qgserror.cpp
   qgsexpression.cpp
+  qgsexpressionparser.cpp
   qgsexpression_texts.cpp
   qgsexpressionfieldbuffer.cpp
   qgsfeature.cpp
@@ -368,7 +369,7 @@ ENDIF (QT_MOBILITY_LOCATION_FOUND OR Qt5Positioning_FO
 
 ADD_FLEX_FILES(QGIS_CORE_SRCS qgsexpressionlexer.ll)
 
-ADD_BISON_FILES(QGIS_CORE_SRCS qgsexpressionparser.yy)
+#ADD_BISON_FILES(QGIS_CORE_SRCS qgsexpressionparser.yy)
 
 SET(QGIS_CORE_MOC_HDRS
 
@@ -812,9 +813,9 @@ ENDIF (QT_MOBILITY_LOCATION_FOUND)
 TARGET_LINK_LIBRARIES(qgis_core
   ${QT_QTMAIN_LIBRARY}
   ${QT_QTXML_LIBRARY}
-  ${QT_QTCORE_LIBRARY}
-  ${QT_QTGUI_LIBRARY}
-  ${QT_QTNETWORK_LIBRARY}
+  ${QT_QTCORE_LIBRARY} ${QT_QTCORE_LIB_DEPENDENCIES}
+  ${QT_QTGUI_LIBRARY} ${QT_QTGUI_LIB_DEPENDENCIES}
+  ${QT_QTNETWORK_LIBRARY} ${QT_QTNETWORK_LIB_DEPENDENCIES}
   ${QT_QTSVG_LIBRARY}
   ${QT_QTWEBKIT_LIBRARY}
 
